src/blocks/timewarrior.rs

```diff
@@ -0,0 +1,205 @@
+//! Time and information of the current timewarrior task
+//!
+//! Clicking left mouse stops or resumes the task
+//!
+//! # Configuration
+//!
+//! Key | Values | Default
+//! ----|--------|--------
+//! `interval` | Update interval in seconds | `30`
+//! `format` | A string to customise the output of the block. See placeholders. | `" $icon {$elapsed\|}"`
```

Comment:
```suggestion
//! `format` | A string to customise the output of the block. See placeholders. | <code>" $icon {$elapsed &vert;}"</code>
```

Unfortunately, `\|` renders as `\|`...